debug.properties

```diff
@@ -1 +1,5 @@
-apiUrl=https://mv-mobile-dev.azurewebsites.net/api/v1/
\ No newline at end of file
+apiUrl=https://mv-mobile-dev.azurewebsites.net/api/v1/
+
+guideUrl=http://monitorizare-vot-ghid.azurewebsites.net/
+organisationWebUrl=https://code4.ro/
+serviceCenterPhoneNumber=0800 080 200
```

Comment:
I think you can remove the spaces from the phone number.

Reply:
> I think you can remove the spaces from the phone number.

done :)